basics/basic-types.md

```diff
@@ -23,7 +23,7 @@ om programmet är kompilerat för ett 32-bitar eller ett 64-bitars system.
 |`real`   | >= 64-bit (generellt 64-bit, men 80-bit på Intel x86 32-bit)
 
 Prefixet `u` betecknar en *osignerad* typ. `char` översätter till
-UTF-8 tecken, `wchar` används i UTF-16 strängar och `dchar`
+ett UTF-8 tecken, `wchar` används i UTF-16 strängar och `dchar`
```

Comment:
That's the correct translation, but I would say that the English text is not consistent.